website/docs/r/branch_protection_v3.html.markdown

```diff
@@ -78,6 +78,7 @@ The following arguments are supported:
 * `branch` - (Required) The Git branch to protect.
 * `enforce_admins` - (Optional) Boolean, setting this to `true` enforces status checks for repository administrators.
 * `require_signed_commits` - (Optional) Boolean, setting this to `true` requires all commits to be signed with GPG.
+* `required_linear_history` - (Optional) Boolean, setting this to `true` enforces a linear commit Git history, which prevents anyone from pushing merge commits to a branch
```

Comment:
Can we move this edit to `website/docs/r/branch_protection.html.markdown` instead of the deprecated `_v3` resource?

Reply:
Done 👍